.evergreen/tasks.yml

```diff
@@ -411,12 +411,12 @@ tasks:
         vars:
           compass_distribution: compass
       - func: test-packaged-app
-        vars:
-          mongodb_version: 4.4.x
-          mongodb_use_enterprise: yes
+        vars: 
+          mongodb_version: '4.4.x'
+          mongodb_use_enterprise: 'yes'
```

Comment:
Super small NIT: we don't need to quote the versions and these other vars here, they are strings regardless

Reply:
This file is built automatically and we need theis quotes upgrade to handle a special character here:`mongodb_version: '>6.0.0-rc0'`.